tests/unit/disco/test_actions.js

```diff
@@ -41,6 +45,13 @@ describe('disco/actions/getDiscoResults', () => {
       .toThrow(/errorHandlerId is required/);
   });
 
+  it('requires platform', () => {
+    const params = defaultParams();
+    delete params.platform;
+    expect(() => getDiscoResults(params))
```

Comment:
I thought we had to make things a little bit more explicit? Something like:

```
expect(() => {
  // func
}).toThrow();
```

Reply:
Indeed! This was copypasta. I'll fix this one and the one I copied it from.